README.md

```diff
@@ -569,12 +571,17 @@ Accounts are the core unit of authentication in the Ripple Network. Each account
 
 Randomly generate keys for a potential new Ripple account.
 
+<!-- <div class='multicode'> -->
+
+*REST*
```

Comment:
`*REST*` shows up in the README and it's just taking up space. So it would be cool to just add:
`<!-- *REST* -->`